crates/solvers/src/domain/order.rs

```diff
@@ -98,11 +98,13 @@ impl<'a> NonLiquidity<'a> {
 #[derive(Clone, Copy, Debug)]
 pub struct Reward(pub f64);
 
+/// Interaction that can be executed as part of an order's pre- or
+/// post-interactions.
 #[derive(Debug, Clone)]
-pub struct CustomInteraction {
+pub struct OrderInteraction {
```

Comment:
naming nit: I would call this `Interaction`. Just because it already is in the order module. Personally, I slightly prefer `order::Interaction` over `order::OrderInteraction`.

Reply:
I thought it had to be changed because we need 2 different interaction types (1 for custom interactions and 1 for pre interactions) but since they are in different scopes we can get away without renaming anything.